website/docs/concepts/router.md

```diff
@@ -164,11 +173,11 @@ fn switch(routes: &Route) -> Html {
 }
 ```
 
-Linking to a specific post is as easy as passing the variant to `Link`:
-
-```rust ,ignore
-<Link<Route> to={Route::Post { id: "new-yew-release".to_string() }}>{ "Yew v0.19 out now!" }</Link</Route>>
-```
```

Comment:
This is misplacement and causes confusion, the `<Link>` element is not introduced anywhere prior to this. It's moved below into the "Navigation" section.